mkosi

```diff
@@ -1953,17 +1942,26 @@ def install_centos(args: CommandLineArguments, root: str, do_run_build_script: b
 
     packages = ['centos-release']
     packages += args.packages or []
-    if args.bootable:
-        packages += ["kernel", "systemd-udev", "binutils"]
     if do_run_build_script:
         packages += args.build_packages or []
+    if args.release == "8":
+        if args.bootable:
+            packages += ["kernel", "systemd-udev", "binutils"]
+    else:
+        if args.bootable:
+            packages += ["kernel", "systemd", "binutils"]
 
-    repos = args.repositories or default_repos
     if args.distribution == Distribution.centos_epel:
-        repos += ["epel"]
+        default_repos += ["epel"]
         packages += ["epel-release"]
```

Comment:
I think this can be reverted as well.

Reply:
This was reverted, I also changed the name of the variable that the return of install_centos_ is put into too match

Reply:
Nevermind during the merge I had removed the line "repos = args.repositories or default_repos" accidentally which was causing me confusion, it has been reverted to the way it was originally.